src/FacadeAccountCreation/FacadeAccountCreation.Core/Models/CreateAccount/ProducerType.cs

```diff
@@ -1,3 +1,5 @@
+using System.Diagnostics.CodeAnalysis;
```

Comment:
I think no need of adding this here! Unless you adding [ExcludeFromCodeCoverage] but not valid on this enum type.